tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
+  "compilerOptions": {
+    "allowJs": false,
+    "allowSyntheticDefaultImports": true,
+    "jsx": "react-native",
+    "module": "esnext",
+    "moduleResolution": "node",
+    "noImplicitAny": false,
+    "noImplicitReturns": true,
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "outDir": "dist",
+    "preserveConstEnums": true,
+    "rootDir": "lib",
+    "skipLibCheck": true,
+    "sourceMap": true,
+    "target": "esnext"
+  },
+  "filesGlob": ["types/index.d.ts", "lib/*.tsx"],
+  "types": ["react", "react-native", "jest"],
```

Comment:
You do not actually have to define this key. When removed, typescript will search everything that's in `node_modules/@types` directory (which is `react`, `react-dom` types and so on)